index.html

```diff
@@ -1528,28 +1540,103 @@
-            countries, sometime along with the former. Persian and Urdu mostly
-            use the third category.</p>
+      <section id="h_naming_conventions">
+	<h4><a href="#h_naming_conventions">Preferred Terminology</a></h4>
+	<p>Before entering this section in  we need to introduce few preferred terminological
+	  conventions for disambiguation and simplicity.</p>
+
+	<ul>
+	  <li><strong><span class="qterm">European Numerals</span></strong> are
+	    0, 1, 2, 3, 4, 5, 6, 7, 8, 9. They are also referred to as <span class="qterm">Western
+	      Arabic Numerals</span> or simply as <span class="qterm">Arabic Numerals</span>. Although
+	    these are terminologically correct terms, to avoid confusions we will refrain from using
+	    these phrases to refer to these numerals. <span class="qterm">European Numerals</span> or
+	    <span class="qterm">ASCII Digits</span> are used instead;
+	  </li>
+	  <li><strong><span class="qterm">Arabic-Indic Numerals</span></strong> are <bdi>٠</bdu>, <bdi>١</bdi>, <bdi>٢</bdi>,
+              <bdi>٣</bdi>, <bdi>٤</bdi>, <bdi>٥</bdi>, <bdi>٦</bdi>, <bdi>٧</bdi>, <bdi>٨</bdi>,
+              <bdi>٩</bdi>;</li>
+          <li><strong><span class="qterm">Eastern Arabic-Indic Numerals</span></strong> are ۰, ۱, ۲, ۳, ۴, ۵, ۶, ۷, ۸, ۹;</li>
+          <li><strong>Extended Arabic-Indic Numerals</strong> same as Eastern Arabic-Indic Numerals.</li>
+	  <li><strong>Western Arabic Numerals</strong> same as European Numerals;</li>
+	  <li><strong>Eastern Arabic Numerals</strong> is used to refer to both Arabic-Indic and
+	    Eastern Arabic-Indic Numerals. Should be avoided due to ambiguity;</li>
+	  <li><strong>Indic Numerals</strong> should be avoided to refer to either of Arabic-Indic
+	    or Eastern Arabic-Indic numerals.</li>
+          <li><strong>Digit</strong>, <strong>Numeral digit</strong>, and <strong>Numeral</strong>
+            are used as synonyms.</li>
+	</ul>
+      </section>
+
+        <section id="h_Families_of_Numerals">
+          <h4><a href="#h_Families_of_Numerals">Families of Numerals</a></h4>
+          <p>There are three families of numerals used with languages using the Arabic Script.
+	    The first is known as European Numerals; the second family is Arabic-Indic Numerals.
+	    The third is Eastern Arabic-Indic Numerals. The following table, based on a similar table in
+	    [[W3-ARAB-MATH]] exhibits these three families:</p>
+
+	  <table border="1">
+	    <thead>
+	      <tr>
+		<th>Family</th>
+		<th>Unicode Range</th>
+		<th colspan="10">Numeral Digits</th>
+		<th>Regions in Use</th>
+	      </tr>
+	    </thead>
+	    <tbody>
+	      <tr>
+		<th>Arabic-Indic</th>
+		<td>U+0660..U+0669</td>
+		<td>٠</td>
+		<td>١</td>
+		<td>٢</td>
+		<td>٣</td>
+		<td>٤</td>
+		<td>٥</td>
+		<td>٦</td>
+		<td>٧</td>
+		<td>٨</td>
+		<td>٩</td>
+		<td>Arabic-speaking countries of the Middle-East, eastern Arabic-speaking countries; e.g. Egypt.</td>
+	      </tr>
+
+	      <tr>
+		<th>European</th>
```

Comment:
Moving this row to the top or bottom would put similar characters (AI and EAI) next to each other.

Putting it on top would make the table sorted by Unicode codepoints.